apps/sms/test/unit/contacts_test.js

```diff
@@ -527,10 +527,10 @@ suite('Contacts', function(done) {
     });
 
     test('The mozContacts find() call returned an error', function(done) {
-      Contacts.findByPhoneNumber('callonerror').then(
-        () => done(new Error('The promise should be rejected.')),
-        () => done()
-      );
+      Contacts.findByPhoneNumber('callonerror').then((results) => {
```

Comment:
why not but I feel we need your other patch that has not landed yet and I think this patch needs to go in asap. Do you think it's good enough if an integration tests is done in a follow-up patch?